test/object-store/CMakeLists.txt

```diff
@@ -93,7 +93,32 @@ endif()
 target_link_libraries(ObjectStoreTests Catch2::Catch2 ObjectStore RealmFFIStatic)
 enable_stdfilesystem(ObjectStoreTests)
 create_coverage_target(generate-coverage ObjectStoreTests)
-add_bundled_test(ObjectStoreTests)
+
+# add_bundled_test(ObjectStoreTests)
+add_labeled_test(ObjectStoreTests-local objstore-local ObjectStoreTests "~[baas]")
```

Comment:
could we preserve the original name and only append the suffix for baas just in case it'd split one day?

Reply:
Using the `objstore-local` vs `objstore-baas` labels (like it currently does) makes it easy to rename these tests without requiring any changes to the `evergreen/config.yml` file. You also get a nice summary at the end of the `ctests` run that shows the timing for each label.